extensions/amp-date-display/0.2/amp-date-display.js

```diff
@@ -55,13 +58,15 @@ class AmpDateDisplay extends PreactBaseElement {
             const container = document.createElement('div');
             container.appendChild(rendered);
 
-            return createElement(RenderDomTree, {
-              'dom': container,
-              'host': host,
-            });
+            return <RenderDomTree dom={container} host={host} />;
           });
-        const asyncRender = createElement(AsyncRender, null, domPromise);
-        return createElement(Fragment, null, children, asyncRender);
+
+        return (
+          <Fragment>
+            {children}
+            <AsyncRender>{domPromise}</AsyncRender>
```

Comment:
Not related to this PR specifically, but feels odd to have `domPromise` passed as a part of `children[]`. Maybe this should be a normal prop?